frontend/DashboardPage.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
+import LinkItem from '../components/LinkItem';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Avatar, Typography } from '@material-ui/core';
+import MailIcon from '@material-ui/icons/Mail';
+import VideoCallIcon from '@material-ui/icons/VideoCall';
+import SearchIcon from '@material-ui/icons/Search';
+import EditIcon from '@material-ui/icons/Edit';
+import InfoIcon from '@material-ui/icons/Info';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+
+export type MailProps ={
+    profilePic: string;
+    name: string;
+}
+
+type DashboardItems = {
+    icon: JSX.Element;
+    optionsName: string;
+}
+
+type Maillist = {
+    profile: JSX.Element;
+    name: string;
+    message: string;
+    dateReceived: string;
+
+}

```

Comment:
I think this type name is misleading. The values describe a single item, but `Maillist` describes a list. The L should be capitalized.